### PR TITLE
Generell update of QGIS Configuration -> Options

### LINKED ISSUE #7585: Generell update of QGIS Configuration -> Options
There are several points in this chapter (en/docs/user_manual/introduction/qgis_configuration.html?options) where the screenshot and description need to be updated.

Page URL:
qgis3 24_options_update

### LINKED ISSUE #7694: Split Options Rendering tab into more contextual and less scarying pages (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#49444
Author: @nyalldawson
QGIS version: 3.28

**Split Options Rendering tab into more contextual and less scarying pages**

### PR Description:
Supersedes https://github.com/qgis/QGIS/pull/49437 by instead moving the options into separate pages

image

image

image

Note: unlike in #49437 I've left the antialias setting in the general rendering page as this setting applies to more than just vector layers.

### Commits tagged with [need-docs] or [FEATURE]
"\n\n\n\n Remove some rendering options from settings dialog\n\nRemoves\n\n- Use render caching option\n- Render layers in parallel option\n\nThese two options are enabled by default and have been for many\nmany years without issues. Now users are more likely to run\ninto issues by disabling these options, and there's really\nno good reason why they should do so.\n\nAlso remove the \"show map canvas refresh in log\" option, which\nis a developer focused option which doesn't need to be exposed\nwidely.\n\n(all options remain available through the advanced settings\npanel for users who REALLY REALLY want to change them)"

### PR DESCRIPTION
**Pull request by PhilipAlbrecht**
Generell update of QGIS Configuration -> Options
Fixes #7585 Fixes #7694
-Texts
- Screenshots

Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -1049,7 +1053,34 @@ https://doc.qt.io/qt-5.9/qnetworkproxy.html#ProxyType-enum
...
 .. _locator_options:
 
-Locator Settings
+GPS
+---
+
+GPSBabel
+........
+
+GPSBabel converts waypoints, tracks, and routes between popular GPS receivers
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_configuration.rst:1062]:
```suggestion
`GPSBabel <https://www.gpsbabel.org/>`_ converts waypoints, tracks, and routes between popular GPS receivers 
```

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 2]
```diff
@@ -1049,7 +1053,34 @@ https://doc.qt.io/qt-5.9/qnetworkproxy.html#ProxyType-enum
...
+GPS
+---
+
+GPSBabel
+........
+
+GPSBabel converts waypoints, tracks, and routes between popular GPS receivers 
+such as Garmin or Magellan and mapping programs like Google Earth or Basecamp. 
+Literally hundreds of GPS receivers and programs are supported.
+First you have to define the :guilabel:`Path to GPSBabel`
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_configuration.rst:1065]:
```suggestion
First you have to define the :guilabel:`Path to GPSBabel` binaries
```
or executable?

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 3]
```diff
@@ -1199,7 +1231,7 @@ You can specify:
 * under :guilabel:`Run and Debug`
 
   * |unchecked| :guilabel:`Enable Object Inspector (switching between tabs may
-    be slow)`: Enable the object inspector.
+    be slow)`: Enable Object Inspector.
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_configuration.rst:1234]:
```suggestion
    be slow)`
```
considering that we do not explain anything here...

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 4]
```diff
@@ -964,32 +964,36 @@ section.
...
 -----------------------
 
 In the :guilabel:`Authentication` tab you can set authentication configurations
 and manage PKI certificates. See :ref:`authentication_index` for more
 details.
 
+To manage authentications, you can use the list of tools next to the frame, ie:
+* |symbologyAdd| :guilabel:`Add new authentication configuration`
+* |symbologyRemove| :guilabel:`Remove selected authentication configuration`
+* |editSelected| :guilabel:`Edit selected authentication configuration`
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_configuration.rst:983]:
```suggestion

* |symbologyAdd| :sup:`Add new authentication configuration`
* |symbologyRemove| :sup:`Remove selected authentication configuration`
* |symbologyEdit| :sup:`Edit selected authentication configuration`
```
missing blank line and, these are tooltips not labels of icons

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 5]
```diff
@@ -1049,7 +1053,34 @@ https://doc.qt.io/qt-5.9/qnetworkproxy.html#ProxyType-enum
...
+
+GPSBabel converts waypoints, tracks, and routes between popular GPS receivers 
+such as Garmin or Magellan and mapping programs like Google Earth or Basecamp. 
+Literally hundreds of GPS receivers and programs are supported.
+First you have to define the :guilabel:`Path to GPSBabel`
+Then you have to choose or to add a device. You can add and delete device paths via
+|symbologyAdd| :guilabel:`Add new path` or |symbologyRemove| :guilabel:`Remove path`.
+After that you can define different parameters:
+
+.. figure:: options_gpsbabel.png
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_configuration.rst:1070]:
```suggestion
.. figure:: img/options_gpsbabel.png
```

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 6]
```diff
@@ -2174,6 +2207,8 @@ in the QGIS user profile.
    :width: 1.2em
 .. |editPaste| image:: /static/common/mActionEditPaste.png
    :width: 1.5em
+.. |editSelected| image:: /static/common/symbologyEdit.png
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_configuration.rst:2210]:
```suggestion
.. |symbologyEdit| image:: /static/common/symbologyEdit.png
```
Let's use the existing substitution

**Review by DelazJ — COMMENTED** [review]
Hi @PhilipAlbrecht 
I have few suggestions below on the text. Also, is the empty line in the "Authentication" screenshot normal?

**Comment by DelazJ** [comment]
This might as well be affected by #7694

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 7]
```diff
@@ -641,7 +641,9 @@ These properties let you set:
   * :guilabel:`Double click action in legend` |selectString|. You can either
     'Open layer properties', 'Open attribute table' or 'Open layer styling dock'
     with the double click.
-  * |checkbox| :guilabel:`Display classification attribute names` in the Layers
+  * |checkbox| :guilabel:`Show feature count for newly added layers` counts the objects of 
+    a newly added layer and writes them in brackets behind, in the layers panel 
+  * |checkbox| :guilabel:`Display classification attribute names` in the layers
```
[the remark this answers is not in the document]
Comment (PhilipAlbrecht) [reply docs/user_manual/introduction/qgis_configuration.rst:646]: what do you mean with this link?

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 8]
```diff
@@ -732,19 +736,15 @@ You can also import or export scales from/to a ``.XML`` file. Note that you
 still have the possibility to remove your changes and reset to the predefined
 list.
 
-
-.. index:: Digitizing configuration
-.. _digitizing_options:
```
[the remark this answers is not in the document]
Comment (PhilipAlbrecht) [reply docs/user_manual/introduction/qgis_configuration.rst:737]: was a mistake. reentered it

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 9]
```diff
@@ -365,20 +365,20 @@ Use a known coordinate to control if your definition is accurate.
...
+Data Sources settings
 ---------------------
 
 .. _figure_data_sources_settings:
 
 .. figure:: img/options_data_sources.png
    :align: center
 
-   Data Sources Settings in QGIS
+   Data Sources settings in QGIS
```
[the remark this answers is not in the document]
Comment (PhilipAlbrecht) [reply docs/user_manual/introduction/qgis_configuration.rst:376]: definitely, just took it out

File: docs/user_manual/introduction/qgis_configuration.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/user_manual/introduction/qgis_configuration.rst:646]: I meant the "show feature..." is being described there.
Same conflict with #7697 and changes at line 718-721

File: docs/user_manual/introduction/qgis_configuration.rst (hunk 8 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/user_manual/introduction/qgis_configuration.rst:737]: Doesn't seem you pushed changes other than my "ready-to-merge" suggestions. These lines are still removed in the latest version and the "in QGIS" text for image caption is still there.

**Pull request merged**